src/ngraph/frontend/onnx_import/core/value_info.hpp

```diff
@@ -67,7 +72,7 @@ namespace ngraph
             ValueInfo& operator=(ValueInfo&&) = delete;
 
             const std::string& get_name() const { return m_value_info_proto->name(); }
-            const Shape& get_shape() const { return m_shape; }
+            const PartialShape& get_partial_shape() const { return m_partial_shape; }
```

Comment:
I think we can retain the name `get_shape` and just change the function to return `PartialShape`s.
In some cases the shape will be fully static and we can use `input.get_shape().is_static()` to check.
I think this will make for a cleaner API.